src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -151,6 +195,99 @@ public ExecutableSequence step() {
     return eSeq;
   }
 
+  /**
+   * Recompute weights for all methods under test at regular intervals.
```

Comment:
I don't see anywhere that the documentation references GRT.  Without documentation of the intended effect, it is difficult for readers to understand this or to validate that it works as intended.